src/praisonai-agents/praisonaiagents/mcp/mcp.py

```
@@ -140,7 +140,7 @@ class MCP:
         ```
     """
     
-    def __init__(self, command_or_string=None, args=None, *, command=None, timeout=60, debug=False, **kwargs):
+    def __init__(self, command_or_string=None, args=None, *, command=None, timeout=60, debug=False, transport="auto", **kwargs):
```

Pylint (3.3.7)

[refactor] Too many arguments (7/5)

(R0913)